src/stream/gst/utils.rs

```diff
@@ -0,0 +1,19 @@
+pub fn is_gstreamer_plugin_available(plugin_name: &str, min_version: &str) -> bool {
+    // reference: https://github.com/GStreamer/gstreamer/blob/b4ca58df7624b005a33e182a511904d7cceea890/tools/gst-inspect.c#L2148
+
+    match gstreamer::init() {
+        Ok(_) => {}
+        Err(error) => println!("Error! {error}"),
+    }
+
+    let mut version = [0, 0, 0];
+    version
+        .iter_mut()
+        .zip(min_version.split('.').map(|s| s.parse().unwrap_or(0u32)))
+        .map(|(a, b)| *a = b)
+        .for_each(drop);
+
+    let registry = gstreamer::Registry::get();
+    registry.lookup_feature(plugin_name).is_some()
+        && registry.check_feature_version(plugin_name, version[0], version[1], version[2])
```

Comment:
```suggestion
    use semver::Version;
    
    let version = Version::from_str(min_version).unwrap();
    let registry = gstreamer::Registry::get();
    registry.lookup_feature(plugin_name).is_some()
        && registry.check_feature_version(
            plugin_name,
            version.major, version.minor, version.patch)
```

Also, a question, what happens if you use check_feature_version directly without lookup feature ? 
It would be helpful to check for feature first and after that check for version and have two different messages to inform the user about the problem. 
With the current code, I don't see the first check necessary.

Reply:
Thanks for the semver suggestion, much cleaner that way.

About the lookup_feature, I'd asked myself the same question, and I have tested that hypothesis and it works, but because the source code for gst-inspect was doing exactly that and I don't know if there are edge cases that have motivated them to double-check that, I've decided to just repeat them instead of investigating it further. Should we really care about it?

Reply:
Okay, now I double-checked and it seems to be used because the APIs are, in fact, different, and in C, they are getting a GstPluginFeature to use in gst_plugin_feature_check_version, which is not needed in the rust API.